### PR TITLE
=clu #17362 Make cluster.joinSeedNodes equivalent to conf seed-nodes

### PR DESCRIPTION
**Pull request by patriknw**
- the difference was in the retry of failed join attempt
- also clarify the documentation

### CODE REVIEW

File: akka-docs/rst/java/cluster-usage.rst [hunk 1]
```diff
@@ -99,11 +101,12 @@ which is attractive when dynamically discovering other nodes at startup by using
...
 supposed to be the first seed node, and that should be placed first in parameter to ``joinSeedNodes``.
 
-Unsuccessful join attempts are automatically retried after the time period defined in 
-configuration property ``retry-unsuccessful-join-after``. When using ``seed-nodes`` this
-means that a new seed node is picked. When joining manually or programatically this means 
-that the last join request is retried. Retries can be disabled by setting the property to 
-``off``.
+Unsuccessful attempts to contact seed nodes are automatically retried after the time period defined in 
+configuration property ``seed-node-timeout``. Unsuccessful attempt to join a specific seed node is
+automatically retried after the configured ``retry-unsuccessful-join-after`. The retries means that it
```
Comment (rkuhn) [inline akka-docs/rst/java/cluster-usage.rst]: “Retrying means …”

File: akka-docs/rst/java/cluster-usage.rst [hunk 2]
```diff
@@ -99,11 +101,12 @@ which is attractive when dynamically discovering other nodes at startup by using
...
+tries to contact all seed nodes and then join the node that answers first. The node that is first
```
Comment (rkuhn) [inline akka-docs/rst/java/cluster-usage.rst]: “… and then join*s* …”

Comment (rkuhn) [inline akka-docs/rst/java/cluster-usage.rst]: “The first node in …”

**Comment by rkuhn** [comment]
LGTM

**Comment by drewhk** [comment]
LGTM

File: akka-docs/rst/java/cluster-usage.rst (hunk 2 above)
Comment (patriknw) [inline akka-docs/rst/java/cluster-usage.rst]: thanks

**Comment by patriknw** [comment]
Refs #17362

**Pull request merged**